src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -26,11 +29,14 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        ArgumentMultimap argMutlimap = ArgumentTokenizer.tokenize(args, PREFIX_ADDRESS);
-        Optional<Address> address = Optional.empty();
-        if (argMutlimap.getValue(PREFIX_ADDRESS).isPresent()) {
-            address = Optional.of(ParserUtil.parseAddress(argMutlimap.getValue(PREFIX_ADDRESS).get()));
-        }
+        ArgumentMultimap argMutlimap = ArgumentTokenizer.tokenize(args, PREFIX_ADDRESS, PREFIX_TAG);
+        Optional<Address> address =
+                !argMutlimap.getValue(PREFIX_ADDRESS).isPresent() ? Optional.empty()
```

Comment:
Is if/else preferred over ternary operators? I'm not sure, need to check on this.

Reply:
Please follow the formats:
alpha = (aLongBooleanExpression) ? beta : gamma;
alpha = (aLongBooleanExpression)
        ? beta
        : gamma;